app/src/main/java/com/example/tasky/agenda_details/presentation/AgendaDetailsViewModel.kt

```diff
@@ -3,11 +3,10 @@ package com.example.tasky.agenda_details.presentation
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import com.example.tasky.agenda_details.data.model.EventDetails
-import com.example.tasky.agenda_details.data.model.PhotoType
+import com.example.tasky.agenda_details.data.model.Photo
```

Comment:
Here, you can see the architectural violation, since presentation should not have a dependency on data. With this architecture, dependencies always point inwards:
presentation -> domain <- data

Reply:
oh no! good catch, I have to keep an eye out for this